feature/system/management_vrf/README.md

```diff
@@ -0,0 +1,85 @@
+# Test plan
+
+## Summary
+
+Test a non default VRF as a VRF for managing the DUT.
+
+## Lab infrastructure
+
+This test plan requires that the DUT is managed by the lab infrastructure in the following way:
+
+![lab_infra](lab_infra.png)
+
+*   All DUT OOB interfaces are L3 with IPv6 addresses connected to lab infrastructure.
+*   Two DUT ports (say **port-11** and **port-12**) are L3 with IPv6 addresses connected to lab infrastructure.
+*   The DUT can be managed at an IPv6 address hosted by a loopback interface, and **via port-11, port-12 or OBB ports**. The loopback interface should host only one IPv6 address. (so that later the test can extract that IPv6 address and move it to another specified loopback interface in a management VRF).
+*   DUT port-1 and port-2 (L3, no vlan) are with IPv6 addresses and are connected to ATE port-1 and port-2, so that the test can exercise/validate some routing protocols (e.g. BGP) without touching lab env infrastructure.
```

Comment:
Your test plan should explicitly state how BGP is configured. Otherwise automation author will do whatever it fit's his need. and not what we want. Expetially if he would be form Arista/Cisco. Hiding OpenConfig violation.